content/sensu-go/5.8/sensuctl/reference.md

```diff
@@ -320,6 +320,15 @@ sensuctl check info check-cpu --format wrapped-json
 In addition to the standard operations, commands may support subcommands or flags that allow you to take special action based on the resource type; the following sections call out those resource-specific operations.
 For a list of subcommands specific to a resource, run `sensuctl TYPE --help`.
 
+### Handling large datasets
+
+When querying sensuctl for large datasets, you use the `--chunk-size` flag with any `list` command to avoid timeouts and improve performance.
+For example, the following command returns the same output as `sensuctl event list` but makes multiple API queries (each for the number of objects specified by `--chunk-size`) instead of one API query for the complete dataset.
+
+{{< highlight shell >}}
+sensuctl event list --chunk-size 20
```

Comment:
20 is probably on the small side... I know `kubectl`'s default is 500. What about just going with 500 in the example?